rpcs3/rpcs3qt/progress_dialog.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "stdafx.h"
+
+#include <QProgressDialog>
+
+#ifdef _WIN32
+#include <QWinTaskbarProgress>
+#include <QWinTaskbarButton>
+#include <QWinTHumbnailToolbar>
+#include <QWinTHumbnailToolbutton>
+#elif HAVE_QTDBUS
+#include <QtDBus/QDBusMessage>
+#include <QtDBus/QDBusConnection>
+#endif
+
+class progress_dialog : public QProgressDialog
+{
+public:
+	progress_dialog(int min, int max, QWidget* parent = (QWidget*)nullptr, Qt::WindowFlags flags = Qt::WindowFlags());
```

Comment:
Why the `(QWidget*)` cast?

Reply:
I just used the same args that are used in QProgressDialog

Reply:
seems like a visual studio thing

Reply:
Well, it's totally unnecessary here.

Reply:
haha true
After Hcorion answered I'll update tomorrow